src/main/resources/application.yaml

```diff
@@ -20,3 +20,8 @@ idam:
   s2s-auth:
     enabled: ${S2S_ENABLED:true}
     url: ${S2S_URL}
+
```

Comment:
Only this hunk contains the fix. Why bother with other changes?

Reply:
service auth library comes in with two feign clients. the "autoconfiged" healthcheck which fails to start the current server itself, and regular api to validate/generate tokens. both clients need `s2s-auth.url` config hence passing it over via docker script too